spec/integration/omniauth-oauth2/Gemfile

```diff
@@ -1,6 +1,9 @@
 source 'http://rubygems.org'
 
 gem 'hashie', path: '../../..'
+# Update omniauth to a pessimistic minor once omniauth releases changes in
+# https://github.com/omniauth/omniauth/pull/977
+gem 'omniauth', git: 'https://github.com/omniauth/omniauth.git', branch: 'master'
```

Comment:
:thinking: Is this part of something else you're working on?

Reply:
see https://github.com/omniauth/omniauth/issues/980, hashie build is currently broken because of omniauth